pkg/executor/scraper/loader.go

```diff
@@ -0,0 +1,10 @@
+package scraper
+
+import (
+	"context"
+)
+
+//go:generate mockgen -destination=./mock_loader.go -package=scraper "github.com/kubeshop/testkube/pkg/executor/scraper" Loader
+type Loader interface {
```

Comment:
Isn't  it `Uploader`? As it's not loading anything :)

Reply:
Or that, yeah, it extracts then uploads